gremconnect/response_test.go

```diff
@@ -77,6 +77,7 @@ var (
 {
     "requestId": "d2476e5b-b2bc-6a70-2647-3991f68ab415",
     "status": {
+        "message": "Some error",
```

Comment:
How about a test case where the message is omitted for backward compatibility.

Reply:
There is. I only added the message for the 401 error, other responses are without message